src/libultra_code/code_801031F0.c

```diff
@@ -1,7 +1,15 @@
 #include "libultra_internal.h"
 #include <global.h>
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_801031F0/func_801031F0.s")
+s32 osAfterPreNMI(void) {
+    return __osSpSetPc(0);
+}
+
+/**
+ * osContStartQuery:
+ * Starts to read the values for SI device status and type which are connected to the controller port and joyport
+ *connector.
```

Comment:
formatting thing, just a space fter the star. Also throughout the codebase Im pretty sure we dont have a linebreak between block comments and the function itself so that would be nice to keep consistent as well